src/Serialization/Address.purs

```diff
@@ -101,14 +104,32 @@ derive newtype instance Eq Slot
 derive instance Newtype Slot _
 derive instance Generic Slot _
 
+instance EncodeJson Slot where
+  encodeJson = undefined -- TODO
+
 instance Show Slot where
   show = genericShow
 
-instance Semigroup Slot where
-  append (Slot s1) (Slot s2) = Slot $ s1 + s2
+-- it is an integer in ogmios
+-- bytestring in plutus
+-- uint32 in csl
+newtype BlockId = BlockId UInt
+
+derive newtype instance Eq BlockId
+derive instance Newtype BlockId _
+derive instance Generic BlockId _
+
+instance EncodeJson BlockId where
+  encodeJson = undefined -- TODO
+
+instance Show BlockId where
+  show = genericShow
+
+-- instance Semigroup Slot where
```

Comment:
Why remove/comment out the `Semigroup` and `Monoid` instances?

Reply:
Forgot to uncomment.